operator/config/overlays/cnsa/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Adds namespace to all resources.
+namespace: ibm-spectrum-scale-csi-driver
```

Comment:
This namespace needs to be `ibm-spectrum-scale-csi`

Reply:
I guess technically it doesn't have to be, but its one less thing CNSA has to worry about in its kustomization.yaml

Reply:
Should we add `namespace.yaml` from CNSA repo to this overlay as well?